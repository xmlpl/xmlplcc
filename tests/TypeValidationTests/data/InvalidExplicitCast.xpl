node[] main() {
  real x = 0;
  integer y = (integer)x;
}
