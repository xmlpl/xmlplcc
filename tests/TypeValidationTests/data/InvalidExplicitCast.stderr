./data/InvalidExplicitCast.xpl:3:11: error: invalid cast from real to integer
./data/InvalidExplicitCast.xpl:error: input contains errors
